### DISCUSSION TITLE
Drag and Drop a file from one Project to another?

### DISCUSSION THREAD

**Discussion by ghenry** [zed-industries/zed#22244] [Feature Requests]
Hi,

Is this possible via a setting or something? Even right-click file, then right-click and paste into another Zed project file navigator view?

Thanks.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
